examples/example.qbs

```diff
@@ -0,0 +1,27 @@
+import qbs
+import "../box2d.qbs" as Box2D
+
+Project {
+    name: "Example";
+
+    Box2D { }
+
+    Product {
+        files: ["main.cpp"];
+        name: "Example";
+        type: "application";
```

Comment:
Ho sorry, didn't realise :-)

Concerning your change proposal, It all make perfect sense to me, I'll change them.

Feel free to change anything you think should be changed.